content/community-projects.md

```diff
@@ -14,14 +14,26 @@ use in your own project.
 
 ## Duffle
 <a href="https://duffle.sh" alt="duffle"><img align="right" src="/img/duffle.svg" width="64px" /></a>
-Duffle is a command line tool that allows you to install and manage CNAB bundles.
+Duffle is the reference implementation of the CNAB specification. It is best for
+installing and managing existing CNAB bundles. If you are making a new bundle,
+the other community tools have extra features designed to make authoring bundles
+easier.
 
 ## Porter
 <a href="https://porter.sh" alt="porter"><img align="right" src="/img/porter.png" width="64px" /></a>
 
-Porter improves the bundle authoring experience by adding bundle dependencies and 
-a declarative experience that understands how to translate CNAB into existing tools
-such as Helm, Azure, etc.
+Porter provides a declarative authoring experience with composable bundles that lets
+you use reuse existing scripts and tools such as Helm, Terraform, Kubernetes, and Azure.
+
+## Docker Apps
+<a href="https://github.com/docker/app" alt="docker"><img align="right" src="/img/docker.png" width="97px" /></a>
```

Comment:
Can we use the last logo ? I can send it to you. Does SVG fit?

Reply:
Sure! Yes please send it to me and I'll use it